app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -88,16 +88,21 @@ form.block-form {
       font-size: 20px;
     }
 
+    .entypo.mail {
```

Comment:
Merge rule `.entypo.mail` with rule on line 81<br>Selector should have depth of applicability no greater than 2, but was 3